src/javascripts/modules/SmoothScrollLink.js

```diff
@@ -0,0 +1,15 @@
+import SmoothScroll from 'smooth-scroll'
+
+export default class SmoothScrollLink {
+  constructor(el) {
+    this.el = el
+    this.href = this.el.getAttribute('href')
+    this.target = document.querySelector(this.href)
+    this.smooth = new SmoothScroll()
+    this.el.addEventListener('click', this.scroll)
+  }
+  
+  scroll = () => {
+    this.smooth.animateScroll(this.target)
+  }
+}
```

Comment:
similar to the points brought up in `ExpandListItem` i would

1. target the parent `ul`and use an array to store the `li` children to maintain the functionality
2. break the constructor into `defineVars()` and `listen()` to keep it clean